README.md

```diff
@@ -2,28 +2,51 @@
 
 **THIS IS WORK IN PROGRESS - PULL REQUESTS & [SUGGESTIONS](https://github.com/aiidateam/aiida-code-registry/issues) HIGHLY WELCOME**
 
-This repository collects configurations of simulation codes on public compute resources for direct setup in AiiDA.
+This repository collects configurations of simulation codes on public compute resources for quick and easy setup in AiiDA.
 
-## Importing computers into AiiDA
+## Using the AiiDA code registry
 
+In the following we'll take the example of [Piz Daint](https://www.cscs.ch/computers/piz-daint/), a HPC system at the Swiss National Supercomputing Centre.
+
+### `verdi computer setup` 
+
+ 1. Navigate to the [`daint.cscs.ch`](./daint.cscs.ch) folder in the GitHub web interface
+ 2. Select the partition you would like to run on, for example [`hybrid`](./daint.cscs.ch/hybrid)
+ 3. Click on the `computer-setup.yaml` file and click on the "Raw" button to get a direct link to the file
+
+Now use this link to set up the computer directly via the `verdi` command line:
 ```
-verdi computer setup --config https:///..../computer-setup.yaml
+verdi computer setup --config https://raw.githubusercontent.com/aiidateam/aiida-code-registry/master/daint.cscs.ch/hybrid/computer-setup.yaml
 ```
+You will be prompted for the computer label (in the following we'll assume you named it `daint-hybrid`).
```

Comment:
I think this line should be moved to the `verdi code setup` section. 

We don't specify the computer label in the code setup, and this is where it will be requested from the user to provide it. The computer setup contains the label, though.

Reply:
Sorry, you're right. I've adapted it